src/nested_pandas/series/accessor.py

```diff
@@ -177,6 +177,33 @@ def query_flat(self, query: str) -> pd.Series:
             return pd.Series([], dtype=self._series.dtype)
         return pack_sorted_df_into_struct(flat)
 
+    def get_flat_series(self, field: str) -> pd.Series:
+        """Get the flat-array field as a Series
+
+        Parameters
+        ----------
+        field : str
+            Name of the field to get.
+
+        Returns
+        -------
+        pd.Series
+            The flat-array field.
+        """
+
+        # TODO: we should make proper missed values handling here
+
+        struct_array = cast(pa.StructArray, pa.array(self._series))
```

Comment:
Not something to think too much about now, but I think these are the sort of operations we should make sure are scalable and performant when we're benchmarking the MVP. I worry about having a huge "source table" and having to do these sorts of operations frequently.

Reply:
This is `typing.cast`, it is just to make type-checker happy

Reply:
```python
In [3]: %%timeit a = pa.array([np.linspace(0, 1, 1_000_000), np.linspace(2, 3, 10_000_000)])
   ...: a = typing.cast(pa.ListArray, a)
   ...:
   ...:
36.3 ns ± 0.124 ns per loop (mean ± std. dev. of 7 runs, 10,000,000 loops each)
```

Reply:
But I absolutely agree about transformations and views, one of the purposes of #15 is to address that

Reply:
Gotcha, yeah makes sense for the cast to be fine. I maybe should have highlighted a different line